doc/include/keck_nires_A.pypeit.rst

```diff
@@ -0,0 +1,30 @@
+.. code-block:: console
+
+    # Auto-generated PypeIt input file using PypeIt version: 1.12.1.dev1+gee479d62d.d20230131
+    # UTC 2023-03-01T23:18:07.752
+    
+    # User-defined execution parameters
+    [rdx]
+        spectrograph = keck_nires
+    
+    # Setup
+    setup read
+    Setup A:
+      dispname: spec
+    setup end
+    
+    # Data block 
+    data read
+     path /Users/westfall/Work/packages/PypeIt-development-suite/RAW_DATA/keck_nires/NIRES
```

Comment:
I made a similar comment on the dev suite. Since the default dither pattern is ABBA, I'm wondering if we should not just dig up the other members of this sequence and add them.

Reply:
There's another dataset that does this in the dev-suite (see the ERIS directory).  It's easy enough for me to change the tutorial to use that dataset.  This dataset provides a good example of how to alter the pypeit file, but the ERIS dataset might be a more typical usage example.  What do you suggest?